uui-components/src/pickers/PickerToggler.tsx

```diff
@@ -199,11 +199,16 @@ function PickerTogglerComponent<TItem, TId>(props: PickerTogglerProps<TItem, TId
         );
     };
 
+    const shouldToggleBody = (e: React.MouseEvent<HTMLDivElement>): boolean => {
+        const isInteractionDisabled = (props.isDisabled || props.isReadonly || isEventTargetInsideClickable(e));
+        const isInputFocusedAndValid = (inFocus && props.value && props.minCharsToSearch);
```

Comment:
Let's better rename to 'shouldOpenWithMinCharsToSearch'

Reply:
done